LoginProvider.swift

```diff
@@ -0,0 +1,66 @@
+//
+//  LoginProvider.swift
+//  Authentication
+//
+//  Created by Gustavo Cairo on 1/13/17.
+//  Copyright © 2017 Wolox. All rights reserved.
+//
+
+import Foundation
+import ReactiveCocoa
+import ReactiveSwift
+import enum Result.NoError
+
+/**
+    Protocol that all users returned by
+    login providers should implement.
+ */
+public protocol LoginProviderUser {}
+
+/**
+    Protocol that all login provider configurations
+    should implement.
+ */
+public protocol LoginProviderConfiguration {}
```

Comment:
Why is this public if the LoginProvider protocol doesn't need it? Does this need to exists?